packages/experiments/src/components/Button/Button.view.tsx

```diff
@@ -28,9 +32,10 @@ export const ButtonView: IButtonComponent['view'] = props => {
       {...buttonProps}
       aria-disabled={disabled}
     >
-      <Slots.stack horizontal as="span" gap={8} verticalAlign="center" horizontalAlign="center">
-        {icon && <Slots.icon />}
-        {content && <Slots.content />}
+      <Slots.stack horizontal as="span" gap={8} verticalAlign="center" horizontalAlign="center" verticalFill>
```

Comment:
will the gap={8} be tokenized eventually? or is the user expected to simply override the stack slot to change it?

Reply:
`gap` is already a prop, so if the user provides it it will simply override this value. A TODO I've called out is to go through the experiments components and 'tokenify' them where it makes sense. (This PR is already huge.)